scripts/get_counts.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+
+# we're using python 3.x style print but want it to work in python 2.x,
+from __future__ import print_function
+import re, os, argparse, sys, math, warnings, subprocess, threading
+
+parser = argparse.ArgumentParser(description="Usage: "
+   "get_counts.py [options] <source-int-dir> <ngram-order> <dest-count-dir>"
+   "e.g.:  get_counts.py data/int 3 data/counts_3"
+   "This script computes data-counts of the specified n-gram order"
+   "for each data-source in <source-int-dir>, and puts them all in"
+   "<dest-counts-dir>.");
+
+parser.add_argument("source_int_dir",
+					help="Specify <source_int_dir> the data-source")
```

Comment:
you ahve a lot too much space as I pointed out by email but I may merge this and we can fix this later, as I'm getting on a plane and want the up-to-date code.